Sources/CodeEditTextView/Controller/STTextViewController.swift

```diff
@@ -127,7 +134,7 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
 
         rulerView = STLineNumberRulerView(textView: textView, scrollView: scrollView)
         rulerView.backgroundColor = useThemeBackground ? theme.background : .clear
-        rulerView.textColor = .systemGray
+        rulerView.textColor = rulerTextColor
```

Comment:
Make the following changes...

```suggestion
        rulerView.textColor = .secondaryLabelColor
```